ee/clickhouse/models/person.py

```diff
@@ -60,10 +71,33 @@ def get_person_distinct_ids():
     return ClickhousePersonDistinctIdSerializer(result, many=True).data
 
 
+def merge_people(target: Person, old_id: int, old_props: Dict) -> None:
+    properties = {}
+    # merge the properties
+    properties = {**old_props, **target.properties}
+
+    update_person_properties(target.pk, properties)
+
+    other_person_distinct_ids = ch_client.execute(
+        GET_DISTINCT_IDS_SQL_BY_ID, {"person_id": old_id, "team_id": target.team.pk}
+    )
+
+    parsed_other_person_distinct_ids = ClickhousePersonDistinctIdSerializer(other_person_distinct_ids, many=True).data
+
+    for person_distinct_id in parsed_other_person_distinct_ids:
+        ch_client.execute(
+            UPDATE_PERSON_ATTACHED_DISTINCT_ID,
+            {"person_id": target.pk, "distinct_id": person_distinct_id["distinct_id"]},
+        )
+
+    ch_client.execute(DELETE_PERSON_BY_ID, {"id": old_id,})
```

Comment:
Not certain how fast these update queries are. They're handled async and could choke up clickhouse if there are too many

Reply:
🤞 best way to test is doing it live